qiskit/providers/backend_compat.py

```diff
@@ -254,7 +344,7 @@ def __init__(
         self,
         backend: BackendV1,
         name_mapping: Optional[Dict[str, Any]] = None,
-        add_delay: bool = False,
+        add_delay: bool = True,
         filter_faulty: bool = False,
```

Comment:
```suggestion
        filter_faulty: bool = True,
```

While I feel that leaving this part as 'False' may potentially confuse the user, is it still preferable to keep it that way?

Reply:
Applied suggestion :)